index.js

```diff
@@ -84,6 +84,192 @@ app.post('/add/:quizName/:count', urlencodedParser, async (req, res) => {
     </html>`)    
 })
 
+////
+////
+////These are the /take routes
+
+app.post('/take', urlencodedParser, async(req, res) => {
+    console.log("This makes sure that this is running properly")  // TEMP
+    
+    var quizName = req.body.userName
+    const inCase = 'Sorry, this quiz does not exist.'
+    if (quizName == '') {
+        quizName = inCase
+    }
+
+    var userName = req.body.quizName
+    if (userName == '') {
+        userName = 'unknown user'
+    }
+
+    console.log(quizName) // TEMP
+    
+    var firstQuestion = await db.get(`${quizName}-question-1`)
+    if (firstQuestion == null) {
+        firstQuestion = "Sorry, this thing your on doesn't exist :("
```

Comment:
This is where you could send the error page and `return`. The error page would be full html and have a button to go back.

Reply:
Go ahead and push those changes - and then click "Resolve conversation" here. No need to reply with an explanation.